<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>509</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Hand Tracking Interface</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>infoLabel_</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="overlayWidget" native="true">
     <property name="styleSheet">
      <string notr="true">background: transparent;
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="SceneGL" name="scene_">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="imageLabel_">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string>Camera Feed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton_">
        <property name="text">
         <string>STOP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="captureButton_">
        <property name="text">
         <string>Start Tracking</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Quitter Le Jeu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SceneGL</class>
   <extends>QOpenGLWidget</extends>
   <header>scenegl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
